homework1/tests/task03.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from homework1.task03 import find_maximum_and_minimum
+
+
+@pytest.mark.parametrize(
+    ["value", "expected_result"],
+    [
+        ("homework1/some_file.txt", (1, 5)),
```

Comment:
If you add an 11 to the file, min and max will be 1 and 5. You shold add more test cases for that function.

Reply:
Fix it.